src/commands/account/add_key/autogenerate_new_keypair/mod.rs

```diff
@@ -30,12 +30,18 @@ impl GenerateKeypair {
 ///Save an access key for this account
 pub enum SaveMode {
     #[strum_discriminants(strum(
-        message = "save-to-keychain   - Save automatically generated key pair to keychain"
+        message = "save-to-keychain         - Save automatically generated key pair to keychain"
     ))]
     ///Save automatically generated key pair to keychain
     SaveToKeychain(self::save_keypair_to_keychain::SaveKeypairToKeychain),
+    #[cfg(target_os = "macos")]
     #[strum_discriminants(strum(
-        message = "print-to-terminal  - Print automatically generated key pair in terminal"
+        message = "save-to-macos-keychain   - Save automatically generated key pair to OS X keychain"
```

Comment:
Let's use "macOS" instead of "OS X" everywhere.

> Apple shortened the name to "OS X" in 2012 and then changed it to "macOS" in 2016 to align with the branding of Apple's other operating systems, iOS, watchOS, and tvOS. After sixteen distinct versions of macOS 10, macOS Big Sur was presented as version 11 in 2020, and macOS Monterey was presented as version 12 in 2021.

https://en.wikipedia.org/wiki/MacOS

Reply:
fixed